h2o-core/build.gradle

```diff
@@ -14,7 +14,6 @@ dependencies {
   compile 'org.javassist:javassist:3.18.2-GA'
   compile "org.apache.commons:commons-math3:3.3"
   compile "commons-io:commons-io:2.4"
-  compile "ai.h2o:google-analytics-java:1.1.2-H2O-CUSTOM"
```

Comment:
FYI: I have to add `org.apache.httpcomponents:httpclient:4.1`, otherwise `testJar` won't compile